wolf-test-utils/src/Wolf/Data/Export/Types/Gen.hs

```diff
@@ -29,22 +33,15 @@ instance GenValid Repo where
         -- The person index consists of (some) people from the global index
         -- and some occur multiple times.
         repoPersonIndex <-
-            let go ix puuid = do
-                    b <- frequency [(3, pure False), (1, pure True)]
-                    if b
-                        then pure ix
-                        else do
-                            a <- genValid
-                            go (addIndexEntry a puuid ix) puuid
+            let gen ix puuid = do
+                    a <- genValid
+                    go (addIndexEntry a puuid ix) puuid
+                go ix puuid = frequency [(3, gen ix puuid), (1, pure ix)]
```

Comment:
Oh, smart, but why did you change this? Seems like a refactor that doesn't belong here.